lib/gui/app/pages/settings/templates/settings.tpl.html

```diff
@@ -43,6 +43,17 @@ <h1 class="title space-bottom-large">Settings</h1>
     </label>
   </div>
 
+  <div class="checkbox">
+    <label>
+      <input type="checkbox"
+        tabindex="8"
+        ng-model="settings.currentData.trim"
+        ng-change="settings.toggle('trim')">
+
+      <span>Trim ext{2,3,4} partitions before writing (raw images only)</span>
```

Comment:
Would it be possible to add a tooltip or something explaining what this actually means / does?